boot/boot_test.go

```diff
@@ -54,24 +55,22 @@ func (s *baseBootSetSuite) SetUpTest(c *C) {
 	s.AddCleanup(restore)
 
 	s.bootdir = filepath.Join(dirs.GlobalRootDir, "boot")
+
+	s.bootloader = bootloadertest.Mock("mock", c.MkDir())
+	bootloader.Force(s.bootloader)
```

Comment:
maybe we should turn this two lines into a mockBootloader method and call it from the two suite that care, then we don't need the extra Force(nil) and also setup is done the same way in all suites